modules/reactor/src/meshgenerators/PeripheralTriangleMeshGenerator.C

```diff
@@ -21,7 +21,8 @@ InputParameters
 PeripheralTriangleMeshGenerator::validParams()
 {
   InputParameters params = MeshGenerator::validParams();
-  params.addRequiredParam<MeshGeneratorName>("input", "The input mesh to be modified.");
+  params.addRequiredParam<std::vector<MeshGeneratorName>>("input",
+                                                          "The input mesh to be modified.");
```

Comment:
If we're potentially taking multiple input meshes, shouldn't the doc string here now reflect that?